packages/apps/origin-backend-irec-app/package.json

```diff
@@ -8,11 +8,11 @@
         "start:dev": "nest start --watch -p tsconfig.json",
         "start:debug": "nest start --debug --watch -p tsconfig.json",
         "start:prod": "node dist/main",
-        "prebuild": "shx rm -rf dist",
+        "prebuild": "rm -rf dist",
         "build": "yarn build:ts",
         "build:ts": "tsc -b tsconfig.json",
         "prettier": "prettier --write --config-precedence file-override './src/**/*'",
-        "clean": "shx rm -rf dist uploads",
+        "clean": "rm -rf dist uploads",
```

Comment:
Will this work cross platform?

Reply:
It works on my machine )) And `shx` does not

Reply:
Does `npx shx` work for you?

Reply:
Let's leave this out of this PR and create a separate PR for this. It's a repo-wide change so we should have a unified approach.

Reply:
Agree, changed to `npx shx rm`
Works fine on Mac, Apple M1